### PR TITLE
Initial UpgradeForm component with basic JS setup.

### PR DESCRIPTION
**Pull request by gabedealmeida**

### CODE REVIEW

**Review by super3 — CHANGES_REQUESTED** [review]
@gabedealmeida Please make the "Plan & Usage" tab is still selected when you are on the upgrade form.

**Comment by gabedealmeida** [comment]
@super3 Great catch!

File: ui/src/components/Usage.vue [hunk 1]
```diff
@@ -252,7 +252,27 @@
 											)
 										}}
 									</td>
-									<td></td>
+									<td>
+										<button
+											v-if="id === 'free-100g'"
```
Comment (montyanderson) [inline ui/src/components/Usage.vue:257]: Can we put this in a computed property?

File: ui/src/components/Usage.vue [hunk 2]
```diff
@@ -252,7 +252,27 @@
...
+											class="btn btn-light btn-sm"
+										>
+											Verify Email
+										</button>
+
+										<router-link
+											to="/app/usage/upgrade-form"
+										>
+											<button
+												v-if="id === 'free-1tb'"
```
Comment (montyanderson) [inline ui/src/components/Usage.vue:268]: This too

**Review by montyanderson — CHANGES_REQUESTED** [review]

**Review by super3 — APPROVED** [review]
Looks good.

File: ui/src/components/Usage.vue (hunk 1 above)
Comment (gabedealmeida) [reply ui/src/components/Usage.vue]: I moved them into two methods because these are inside of the v-for loop

**Review by montyanderson — APPROVED** [review]

**Pull request merged**